pkgs/applications/science/programming/scyther/sources.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub }:
+let
+  versions = map (s: { name = s.rev; } // s) [
```

Comment:
We typically provide only a single version. If users want another version, they could override and pass their own `src` in.